lib/lita/handlers/reviewme.rb

```diff
@@ -116,6 +118,19 @@ def mention_reviewer(response, room: get_room(response))
         response.reply(chat_mention(reviewer, url))
       end
 
+      # Now that we lowercase inputted usernames to make our commands fully case insensitive, we
+      # need to migrate all reviewer lists to make sure all usernames are lowercase, or otherwise
+      # users would not be able to remove usernames that already had uppercase characters in them.
+      def ensure_usernames_are_lowercase(payload)
+        lists = redis.keys("*")
```

Comment:
It's been awhile since I've worked in redis, but this seems a little greedy. Is the `redis` instance already namespaced to only to this handler or something?

Reply:
Yep, exactly. So this would only return keys that begin with `lita:handlers:reviewme:`. You think it's worth a comment above this line noting that nuance?

Reply:
Yes, that sounds useful :)

Reply:
Hurray, thank you! It'll be next week before I can closely look at this. I want to think about it all a little more before merge. Thank you so much for all your work on it! 🎉